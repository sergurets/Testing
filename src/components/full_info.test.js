import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import FullInfo from './full_info';
describe('Link', () => {
  it('should render correctly', () => {
    const output = shallow(
      <FullInfo title="title" release_date="date" genres={['rock']} />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});