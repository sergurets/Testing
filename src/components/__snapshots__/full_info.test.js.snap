// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render correctly 1`] = `
<div
  className="full_info_wrapper"
>
  <img />
  <div
    className="full_info_content"
  >
    <span>
      title
    </span>
    <span
      className=""
    >
      date
    </span>
    <span
      className=""
    />
  </div>
</div>
`;
